packages/react-scripts/config/webpack.config.js

```diff
@@ -56,7 +56,12 @@ const reactRefreshOverlayEntry = require.resolve(
 
 // Some apps do not need the benefits of saving a web request, so not inlining the chunk
 // makes for a smoother build process.
-const shouldInlineRuntimeChunk = process.env.INLINE_RUNTIME_CHUNK !== 'false';
+// const shouldInlineRuntimeChunk = process.env.INLINE_RUNTIME_CHUNK !== 'false';
```

Comment:
Why do we need commented duplication?

Reply:
This was left due to the probability of rolling back in case we will use an approach with env variables and the complexity of CRA project.